File: leetcode2848/solveLeetcode2848.cpp
// class Solution {
// public:
//     int numberOfPoints(vector<vector<int>>& nums) 
//     {
//         vector<int>f(105);
//         for(const auto&num:nums)
//         {
//             ++f[num[0]],--f[num[1]+1];
//         }

//         int ans=0;
//         for(int i=0;i<=100;++i)
//         {
//             f[i+1]+=f[i];
//             ans+=f[i+1]>0;
//         }

//         return ans;
//     }
// };

File: leetcode2848/solveLeetcode2848.go
// func numberOfPoints(nums [][]int) int {
//     f := make([]int, 105)
//     for _, num := range nums {
//         f[num[0]]++
//         f[num[1]+1]--
//     }

//     ans := 0
//     for i := 0; i <= 100; i++ {
//         f[i+1] += f[i]
//         if f[i+1] > 0 {
//             ans++
//         }
//     }

//     return ans
// }